src/components/UserPathLog/Login.js

```diff
@@ -1,41 +1,40 @@
-import Axios from 'axios';
 import React, { useState } from 'react';
-import { useHistory } from 'react-router-dom';
+import { useHistory, Link } from 'react-router-dom';
 // import Registration from '../UserPathLog/Register';
 // import Resetpw from '../UserPathLog/Resetpw';
-import { Link } from 'react-router-dom';
+import './Login.css';
+import { Logo } from '../styleElements/icons/Logo';
+import { Button } from '../styleElements/buttons/Button';
 
-import { UnderConstruction } from '../UnderConstruction';
+import { loginUser } from '../../Redux/userSlice';
+import { useDispatch } from 'react-redux';
+import { unwrapResult } from '@reduxjs/toolkit';
 
 export default function Login() {
   const [username, setUsername] = useState('');
   const [password, setPassword] = useState('');
-  const serverUrl = process.env.REACT_APP_SERVER;
 
   const history = useHistory();
+  const dispatch = useDispatch();
 
-  //this could be changed to an async function
   //this verification route will work if you run the server from index.js on your local machine
   //currently the credentials username:Admin password:password will work
   //a registration route is set up under POST "http://localhost:5000/authentication/register" (also accepts username and pw)
-  function login() {
```

Comment:
the reason the login function wasn't working was because we'd changed how authentication was being done, I took the liberty of just fixing it.  This was totally unrelated to anything you'd written.